index.html

```diff
@@ -495,6 +495,462 @@ <h2>
+                </ol>
+              </li>
+              <li>Else,
+                <ol>
+                  <li>Let |params:LaunchParams| be a new {{LaunchParams}} with
+                  {{LaunchParams/files}} set to a {{FrozenArray}} of
+                  {{FileSystemHandle}}s that correspond to the file paths named
+                  by |files|.
+                  </li>
+                  <li>[=launch a web app=] with |params|.
+                  </li>
+                </ol>
+              </li>
+            </ol>
+          </li>
+        </ol>
+      </section>
+      <section>
+        <h3>
+          Registering file handlers
+        </h3>
+        <p>
+          A user agent SHOULD <dfn>register file handlers</dfn> with the host
+          operating system, consistent with:
+        </p>
+        <ul>
+          <li>The app is exposed in appropriate OS surfaces such as the native
+          file browser and other surfaces that display lists of apps that can
+          handle a given file or file type.
+          </li>
+          <li>The app is a candidate for becoming the system's default handler
+          for associated file types.
+          </li>
+        </ul>
+        <p>
+          A user agent MAY truncate the total set of [=file extensions=] to
+          preserve functionality and prevent abuse. A user agent MAY prevent
+          associations with certain filetypes.
+        </p>
+      </section>
+      <section>
+        <h3>
+          Privacy consideration: Default file handler.
+        </h3>
+        <p>
+          Depending on the operating system capabilities, the [=file handler=]
+          could become a 'default' handler (e.g. handling launches of a given
+          file type automatically) of a given [=file type=] without the
+          explicit knowledge of the user. To protect against possible mis-use,
+          user agents MAY require explicit user consent to begin with the
+          process to [=execute a file handler launch=]. If consent is sought,
+          the user agent SHOULD allow the user to specify that the decision is
+          permanent and if so specified SHOULD remove the web application's OS
+          registration as a file handling entity.
+        </p>
+      </section>
+      <section>
+        <h3>
+          Privacy consideration: Name and icon.
+        </h3>
+        <p>
+          The name and icon of each file handler can be sensitive to privacy
+          and security, as there isn't a specified way for the user to see and
+          confirm these. Due to this, user agents MAY choose to ignore these in
+          favor of alternative strings and icons or fall back on OS defaults.
+        </p>
+      </section>
+      <section class="informative">
+        <h3>
+          Example manifest with file handlers
+        </h3>
+        <p>
+          In the following example, the web application has 3 different file
+          handlers.
+        </p>
+        <aside class="example">
```

Comment:
The example should include the `"launch_type"` member for the sake of completeness.